sponsoring.md

```diff
@@ -0,0 +1,116 @@
+# Sponsoring
+
+After we found a venue the next step is always find sponsors. The reason for this is that we finance the conference only with sponsors and the money from the tickets.
+
+## Sponsors
+
+When we write to sponsors we write them what we offer (Number of participants, topics, number of talks, ...) and attach additionally a standardized cover letter and some promotion slides to the email. We added our latest cover letter to the section [sample letter below.](#sample-letter)
+
+To have an overview of the status of the requests we use a google spreadsheet. At the moment it has the following head lines and information:
+
+| Company name |	Contact name | Contact e-mail |	Addressed on |	Addressed by | Current Status | Additional comment | Invoice sent | Added to website | Tweet sent | Ticket links sent |
+|-------|----|-----|-----|-----|-----|-----|----|----|----|-----|
+| Awesome Company |	Evelyn Berezin | `evelyn@awesome.com` |	23.1.2017 |	Tina | Confirmed - Gold |  | - | yes | - | - |
+| Superduper Company | Edith Clarke | `edith@super.com` |	24.1.2017 |	Tina | Cancelled | Ask them next year again | - | - | - | - |
+| Amazing Corporation | Ada Lovelace | `ada@amazing.com` |	24.1.2017 |	Tina | Sent |  | - | - | - | - |
+| New Startup | Radia Perlman | `radia@startup.com` |	27.1.2017 |	Tina | Confirmed - Silver | Remind them of sponsoring material | 10.2.2017 | yes | - | 10.2.2017 |
+
+We also use `Conditional Formatting` to count the number of sponsors per level and to highlight the status from the requests.
+
+In another tab we calculate the money we get from the sponsors to have a little overview how much money we already collected.
+
+![Sponsor overview](material/sponsor-overview.png)
+
+### Sponsoring plans
+
+#### What do the sponsors get
+
+Depending on the sponsoring level the sponsors get from us different things.
+
+On the one hand they get free tickets and options to buy more tickets (in case we are sold out these tickets are still buyable for the sponsors).
+
+On the other hand they get a lot of opportunities to advertise their company. Depending on the level they may put up posters or booths to talk to attendees. We also mention some in tweets or in any mails we send to our participants.
+
```

Comment:
Readability:

`We also mention some in tweets or in any mails we send to our participants.`
to
`Depending on the level we mention them also in tweets or mails sent to the participants`
?

for me the relation of the `some` was not clear on first read